app/src/main/kotlin/com/wire/android/ui/home/conversations/info/ConversationInfoViewState.kt

```diff
@@ -33,7 +34,9 @@ data class ConversationInfoViewState(
     val conversationDetailsData: ConversationDetailsData = ConversationDetailsData.None,
     val conversationAvatar: ConversationAvatar = ConversationAvatar.None,
     val hasUserPermissionToEdit: Boolean = false,
-    val conversationType: Conversation.Type = Conversation.Type.ONE_ON_ONE
+    val conversationType: Conversation.Type = Conversation.Type.ONE_ON_ONE,
+    val mlsVerificationStatus: ConversationVerificationStatus? = null,
+    val proteusVerificationStatus: ConversationVerificationStatus? = null
```

Comment:
Can it be MLS-verified and Proteus-not-verified at the same time or can be just one of them?
If only one at a time then I would prefer to only have a single field determining status, and another determining protocol, in my opinion it's less confusing, like:
```
val verificationStatus: ConversationVerificationStatus? = null,
val protocol:  ConversationProtocol
```

But it's a minor thing  so I won't fight about it 😄

Reply:
make sense, merged both states into 1